src/css/utilities/fluid-type.css

```diff
@@ -0,0 +1,59 @@
+/**
+  * Fluid typography
+  *
+  * Typography with font size that increases smoothly and linearly between a min and max value
+  * over the viewport width range 360px (XS) to 1500px (2XL)
+  * See concept on https://www.aleksandrhovhannisyan.com/blog/fluid-type-scale-with-css-clamp/
+  *
+  * See google sheet for a plot of the font sizes over the viewport width range 360px (XS) to 1500px (2XL):
+  * https://docs.google.com/spreadsheets/d/1AN9eV_ECXc9MzTZUc10hiLjOHeOCDTutC5tMHOCPqgo/edit?usp=sharing
+  *
+  * Good for use in hero banners and horizontal cards where UI looks better with a smooth transition of font size
+  * Here we offer the sizes from fluid-type-0 to fluid-type-10. Their min and max values at the XS and 2XL breakpoints
+  * are the same as the corresponding modular typography classes (type-0 to type-10) with base font sizes
+  * XS: 1.8rem, MD: 2.1rem, 2XL: 2.3rem
+  */
+
+@utility fluid-type-0 {
+  font-size: clamp(1.8rem, 0.44vw + 1.64rem, 2.3rem);
+}
+
+@utility fluid-type-1 {
+  font-size: clamp(2.1rem, 0.7vw + 1.85rem, 2.9rem);
+}
```

Comment:
Moved fluid type to utilities instead of components. Apparently in TW v4, things like lg:fluid-type-4 (when you use a modifier) stops working if fluid-type is a component.